doc/README.md

```diff
@@ -0,0 +1,81 @@
+# SciPy Documentation
+
+## How to build the docs
+
+To build the html docs for local development, SciPy itself needs to be built so your
+environment needs to be set up for that.  For details on that, see the
+[Contributor Guide](http://scipy.github.io/devdocs/dev/contributor/contributor_toc.html#development-environment)).
+
+Also ensure to initialize and update submodules (this pulls in the SciPy Sphinx
+theme and `numpydoc`):
+```
+git submodule update --init
+```
+
+Now to build both SciPy itself and the docs, use:
+```
+python3 runtests.py --doc html
+```
+
+Alternatively, if you prefer to build SciPy and the docs separately rather
+than use `runtests.py`:
+```
+python setup.py develop  # in the root of the repo
+cd doc && make html
+```
+
+In case the SciPy version found by the above command is different from that of the
+latest commit in the repo, you will see a message like:
+```
+installed scipy 5fd20ec1aa != current repo git version '35fd20ec1a'
+```
+
+This indicates that you're likely picking up the wrong SciPy install, check
+with `python -c "import scipy; print(scipy.__file__)"`.
+
+If the build is successful, you can open it in your browser, e.g. with
+`firefox build/html/index.html`.
```

Comment:
I would say:
```suggestion
`make show` (which will launch your default browser).
```

See https://github.com/scipy/scipy/blob/master/doc/Makefile#L158-L159. But this will only work for `make html-scipyorg`. So if `make html` and `make html-scipyorg` are otherwise equivalent, this is one reason to prefer telling people to do `make html-scipyorg`.

The other is that `make html-scipyorg` is the one that actually gets published, so it's closer to what people will actually see on the website. Sometimes the format/styling can matter, so why not make things more accurately reflect what will be published?

Reply:
added `make show`, thanks